packages/cli/src/__tests__/commands/Studio.test.ts

```diff
@@ -27,89 +28,93 @@ function sendRequest(message: any): Promise<any> {
 }
 
 let studio: Studio
-describe('studio with alternative urls and prisma://', () => {
-  afterEach(() => {
-    // Back to original env vars
-    process.env = { ...originalEnv }
-  })
+// Prisma Studio ignores env vars for overriding engine paths, skipping test for now
+describeIf(!process.env.PRISMA_QUERY_ENGINE_LIBRARY && !process.env.PRISMA_QUERY_ENGINE_BINARY)(
```

Comment:
Should this condition be a helper that denotes the intent?

Reply:
This condition is slightly different in every test, not sure single helper would cut it.